### ISSUE TITLE
Coordinate property missing iOS GoogleMaps in MapView.Marker onPress

### ISSUE THREAD

**Issue by jskidd3** [react-community/react-native-maps#1967]
### Is this a bug report?

Yes

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

Yes

### Environment

react-native-maps@0.17.1
react: 16.0.0
react-native: 0.50.3

Target Platform: iOS 11.1.2 iPhone 6S
Map: GoogleMaps

### Steps to Reproduce

1. Add a MapView.Marker to a MapView
2. Add a `onPress` prop to the MapView.Marker
3. In the callback function the value of the event's `nativeEvent`, the `coordinate` property is missing.

### Expected Behavior

I expected to see a `coordinate` property in the event object.

### Actual Behavior

The `coordinate` property is non-existent. If I remove the map provider and use AppleMaps then the `coordinate` value shows, `GoogleMaps` (our required platform) returns no `coordinate` property.

### Reproducible Demo

https://snack.expo.io/ByIuKY9EM

**Comment by jskidd3** [comment]
If anyone knows a workaround that would allow my app to retrieve the coordinates of a selected marker when it is pressed on GoogleMaps iOS I would be very grateful!

**Comment by rborn** [comment]
@jskidd3 You could wrap the MapView.Marker in a component and "hijack" the onPress to pass the coordinates set as props.

**Comment by jskidd3** [comment]
@rborn I'm fairly new RN, would you mind providing a small code example? Would the onPress of the wrapped component contain the coordinates making this possible? I assumed I'd only be able to get the coordinates from the MapView.Marker onPress.

**Comment by rborn** [comment]
@jskidd3 something like this (**untested/pseudocode**):
~~~
class MyMarker extends Component {
    onPress  = () => {
        this.props.onPress(this.props.coordinates)
    }
    
    render() {
        return <MapView.Marker onPress={this.onPress} coordinates={this.props.coordinates}/>
    }
}
~~~
and in your file you use `MyMarker` instead of `MapView.Marker`.
Makes sense?

**Comment by jskidd3** [comment]
@rborn Doesn't appear to be rendering. Any ideas? Created a snack: https://snack.expo.io/BJFFEhi4z

**Comment by FaBeyyy** [comment]
i have the same error what do you mean by singular? e.nativeEvent.coordinate is undefined for me

**Comment by jskidd3** [comment]
@FaBeyyy That's because in my example I just return `coordinate` from the `onPress` inside my customer marker rather. Here I fixed it for you in this Snack: https://snack.expo.io/B1Rv4jxrz

**Comment by RidaRidss** [comment]
<MapView.Marker
          key={marker.key}
          identifier={marker.id}
          onPress={e => {
            e.stopPropagation();
          }}
          image={marker.Image}
          coordinate={{
            latitude: marker.latitude,
            longitude: +marker.longitude
          }}
        />

**Comment by rafaellincoln** [comment]
@rborn I include this fix on #2011

**Comment by rborn** [comment]
@rafaellincoln so we can close this one?

**Comment by rafaellincoln** [comment]
@rborn yes.

**Comment by rborn** [comment]
Closing as per comments

**Comment by jassonha21** [comment]
@rafaellincoln when will be this PR be included on the update?

**Comment by rborn** [comment]
@jass-trix it's in master merged with #2011 . Use the git version.
